concourse/pipelines/create-deployer.yml

```diff
@@ -413,6 +413,7 @@ jobs:
     serial: true
     plan:
     - get: paas-cf
+      passed: [vpc]
```

Comment:
Actually, thinking about this further, this is bad. A terraform run must always use the latest version of the state file (ie not have it pinned). See https://github.com/alphagov/paas-cf/pull/103 for more details, but the summary is that if the state file is pinned, it can potentially run with an older statefile that won't match the actual state of AWS.